tests/epyccel/recognised_functions/test_numpy_types.py

```diff
@@ -9,9 +9,121 @@
 from test_numpy_funcs import max_float, min_float, max_float32, min_float32,max_float64, min_float64
 from test_numpy_funcs import matching_types
 
-from pyccel.decorators import types
+from pyccel.decorators import types, template
 from pyccel.epyccel import epyccel
 
+def test_mult_numpy_python_type(language):
+
+    def mult_on_array_int8():
+        from numpy import ones, int8
```

Comment:
is there a reason why you import numpy in every function def?, I ve noticed that most tests have the same import statments.